metrics-micrometer/src/test/java/org/axonframework/micrometer/CapacityMonitorTest.java

```diff
@@ -36,39 +38,50 @@
 class CapacityMonitorTest {
 
     @Test
-    void testSingleThreadedCapacity() {
+    void testCapacity() {
```

Comment:
Isn't there still value in the single and multi threaded capacity test?

Reply:
Not sure why this test method was named `testSingleThreadedCapacity ` and the one below `testMultithreadedCapacity`

Reply:
I am guessing it was because of the provided properties the single threaded test did?